minicluster/pom.xml

```diff
@@ -10,8 +10,8 @@
 
   <artifactId>tachyon-minicluster</artifactId>
   <packaging>jar</packaging>
-  <name>Mock Tachyon Cluster</name>
-  <description>Mock Tachyon Cluster for testing</description>
+  <name>Tachyon Mock Cluster</name>
+  <description>Tachyon Mock Cluster for testing</description>
```

Comment:
Since we are doing some renaming, I find it weird that the artifact ID is not the same as the name. (`Mock` vs `Mini`)